src/tests/ftest/ior/intercept_basic.yaml

```diff
@@ -15,39 +15,23 @@ server_config:
 pool:
     mode: 146 # 146 is RW
     name: daos_server
-    # 300 G
-    scm_size: 30000000000
-    nvme_size: 420000000000
+    size: 90%
     svcn: 1
     control_method: dmg
 container:
     type: POSIX
     control_method: daos
 ior:
     client_processes:
-        np_16:
-            np: 16
+        np: 16
     test_file: testFile
-    repetitions: 5
-    iorflags:
-        ssf:
-          flags: "-v -D 600 -w -r"
-          ior_api:
-            - DFS
-            - POSIX
-          transfersize_blocksize:
-            1M:
-              transfer_size: '1M'
-              block_size: '2G'
-              # Expected performance improvement
-              # of write and read for the specific
-              # transfer/block size when using
-              # interception library.
-              write_x: 1.5
-              read_x: 1.5
-          objectclass:
-            oclass_SX:
-              dfs_oclass: "SX"
+    repetitions: 3
```

Comment:
3 repetitions seems to be good enough for these tests, but we can certainly increase for more stability